client/push/push_manager.js

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2017 The Absolute Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * push class
+ */
+
+export default class Push {
+  /**
+   * @constructor
+   */
+  constructor() {
+    /* Token to use for push notification */
+    this._token = 0;
+  }
+
+  /**
+   * send push information to server
+   */
+  addPushToServer(token) {
+    this._token = token;
```

Comment:
This could be adding push subscription to server, how about registerSubscription?
eg) push.registerSubscription(subscription)